assets/i18n/en.json

```diff
@@ -292,6 +292,8 @@
     "I18N_EXPLORATION_RvopsvVdIb0J_TITLE": "Tying Ratios to Actual Numbers",
     "I18N_EXPLORATION_SR1IKIdLxnm1_DESCRIPTION": "Ava got bored from playing the amusement park games, so Kamal created a fun math game. Can you beat Kamal's game? Click on this lesson to find out!",
     "I18N_EXPLORATION_SR1IKIdLxnm1_TITLE": "Recap: Variables",
+    "I18N_EXPLORATION_STARTING_FROM_CHECKPOINT": "Welcome back! Continuing this lesson from where you left off",
+    "I18N_EXPLORATION_STATE_PREVIOUSLY_COMPLETED": "You have already covered this in a previous session.",
```

Comment:
Can you show this in the UI video proofs?

Reply:
Done